tests/test_corpus_creators.py

```diff
@@ -35,3 +35,5 @@ def test_simple_corpus_creator():
 
     corpus = SimpleCorpusCreator()._transform(sample_document)
     assert corpus == 'we are looking for a person to fill this job here are some experience and requirements here are some qualifications customer service consultant entry level'
+    corpus = GensimCorpusCreator()._transform(sample_document)
```

Comment:
I prefer a different test function for each corpus creator.

They can still share the input sample document, though. It could be defined at module level instead of within the test function.